src/main/java/de/qabel/desktop/cellValueFactory/BoxObjectCellValueFactory.java

```diff
@@ -43,9 +44,10 @@ public BoxObjectCellValueFactory(String searchValue) {
         if (bf instanceof BoxFile) {
             switch (searchValue) {
                 case SIZE:
-                    return new ReadOnlyStringWrapper(((BoxFile) bf).getSize().toString());
+                    String fileSizeFormat = FileUtils.byteCountToDisplaySize(((BoxFile) bf).getSize());
```

Comment:
Still a bad name. What does that string contain and what is it used for? It contains the formatted file size. Try to find a name that describes that.